### ISSUE TITLE
There should be a script to handle updating of devices one-by-one for special use-cases

### ISSUE THREAD

**Issue by CameronDiver** [balena-io-projects/staged-releases#19]
<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_b4ab)

**Comment by thgreasi** [comment]
@CameronDiver can you elaborate on this?
Isn't `./set-device-to-a-build.sh` already doing that?

**Comment by CameronDiver** [comment]
No, that would set a single device to a build. Say you want to do a staged rollout of 100 devices, that's 100 calls to that script, with different parameters and optionally waiting for the device to have downloaded the application. 

I know this will be possible via the dashboard when those integrations are present, but until then a script here would help the users which want this feature.

**Comment by thgreasi** [comment]
Oh ok, even though we can somewhat do that already setting tags to the devices of interest and using `./update-test-group.sh`.

**Comment by CameronDiver** [comment]
That would either involve grouping the devices (which may be appropriate, depends on use case) or assigning an individual tag per device, which has the same issue.
